docs/administration/metrics/_index.md

```diff
@@ -0,0 +1,91 @@
+
+In Harbor v2.2 and later you are able to enable metrics in your Harbor [configuration file](../../install-config/configure-yml-file.md). Harbor metrics are available at `<harbor_instance>:<metrics_port>/<metrics_path>` based on your configured values. Harbor exposes metrics for the Harbor database (through the Harbor exporter component), the core pod, and the registry.
+
+For each of these components, the following metrics are exposed
+* Runtime info from the GO library
+* Performance metrics about all API requests in core
+* Number of requests in core in flight
+* Metrics provided by docker distribution itself
+* Some data related to business logic which already exist in Harbor DB
+
+## Harbor Exporter Metrics
+
+The following are metrics pulled from the Harbor database and are available at `<harbor_instance>:<metrics_port>/<metrics_path>`.
+
+{{< table caption="Metrics exposed by the Harbor Exporter" >}}
+Name | Description | Labels | Metric type
+:---------|:------------|:-------|:-------
+`harbor_project_total` |	Total number of public and private projects | public (`true`,`false`) | gauge
+`harbor_project_repo_total` |	Total number of repositories in a project |	public (`true`,`false`),  project_name | gauge
+`harbor_project_member_total` |	Total number of members in a project |	project_name | gauge
+`harbor_project_quota_usage_byte` |	Total used resources of a project |	project_name | gauge
+`harbor_project_quota_byte` |	Quota set in a project |	project_name | gauge
+`harbor_artifact_pulled` |	Number of images pulled in a project |	project_name | gauge
+`harbor_project_artifact_total` | Total number of artifacts type in a project | artifact_type (`IMAGE`, *** ), project_name, public (`true`,`false`) | gauge
+`harbor_health` | Current status of Harbor | | gauge
+`harbor_system_info` | Information about your Harbor instance | auth_mode(`db_auth`, *** ), harbor_version, self_registration(`true`,`false`) | gauge
+`harbor_up`| Running status of Harbor components  | component (`chartmuseum`, `core`, `database`, `jobservice`, `portal`, `redis`, `registry`, `registryctl`, `trivy`) | gauge
+{{< /table >}}
+
+## Harbor Core Metrics
+The following are metrics pulled from the Harbor core pod and are available at `<harbor_instance>:<metrics_port>/<metrics_path>?comp=core`.
+
+{{< table caption="Metrics exposed by Harbor Core" >}}
+Name | Description | Labels | Metric type
+:---------|:------------|:-------|:-------
+`harbor_core_http_inflight_requests` | The total number of requests | | gauge
+`harbor_core_http_request_duration_seconds` | The time duration of the requests | method (`GET`, `POST`, `HEAD`, `PATCH`, `PUT`), operation, quantile | summary
+`harbor_core_http_request_total` | The total number of requests | method (`GET`, `POST`, `HEAD`, `PATCH`, `PUT`), operation | counter
```

Comment:
should add `operation` label

Reply:
@ninjadq, i'm not sure which one you mean, the operation label is listed for harbor_core_http_request_total` and `harbor_core_http_request_duration_seconds`

Reply:
sorry, I mean it should add the values for operation label. Because some values may confuse the users.

Reply:
updated